web/themes/custom/usagov/usagov.theme

```diff
@@ -36,47 +37,53 @@ function children_of_active_menu_item(&$items) {
  * Process block content
  */
 function usagov_preprocess_block(&$variables) {
-  $variables['node'] = \Drupal::routeMatch()->getParameter('node');
+  $node = \Drupal::routeMatch()->getParameter('node');
+  $variables['node'] = $node;
+
+  if ($node->hasField('field_page_type') && $node->get('field_page_type')[0]->target_id) {
```

Comment:
Okay, this is where I'm hitting an error message when I try to log in to the cms: 

Error: Call to a member function hasField() on null in /var/www/web/themes/custom/usagov/usagov.theme

In this context, we need to account for there not being a node. The old arrangement sort of did that by accident.

Reply:
Thanks! I've added the check for $node and removed the debugging code